src/styles/components/page-header/styles.less

```diff
@@ -171,6 +172,80 @@
       & when not (@pod-margin-bottom = null) {
         margin-bottom: -@pod-margin-bottom;
       }
+
+      &-tabs {
+        display: none;
+      }
+
+      &-dropdown {
+        border-top: 1px solid @page-header-navigation-dropdown-border-color;
+        display: block;
+        flex: 1 1 auto;
+        margin-left: @page-header-navigation-dropdown-margin-left;
+        margin-right: @page-header-navigation-dropdown-margin-right;
+
+        &-button {
+          border-radius: 0;
+          height: @page-header-dropdown-menu-button-height;
+          padding-left: @page-header-navigation-dropdown-padding-left;
+          padding-right: @page-header-navigation-dropdown-padding-right;
+          text-align: left;
+          text-transform: none;
+          width: 100%;
+
+          button& {
```

Comment:
Yeah, the `&` is the parent selector so the compiled selector will be `button.page-header-navigation-dropdown-button`